S2VX.Game/Editor/ColorPicker/S2VXColorPicker.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+
+using osu.Framework.Bindables;
+using osu.Framework.Caching;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.UserInterface;
+using osuTK;
+using osuTK.Graphics;
+
+namespace S2VX.Game.Editor.ColorPicker {
+    public class S2VXColorPicker : Container, IHasCurrentValue<Color4> {
+        // Change current value will cause recursive change, so need a record to disable this change.
+        private readonly Cached InternalUpdate = new Cached();
+
+        private readonly BindableWithCurrent<Color4> CurrentColor =
+            new BindableWithCurrent<Color4> { Default = Color4.White };
```

Comment:
Can probably be simplified to new() to follow C# 9

Reply:
Implemented. Also did the same for attributes in other files

Reply:
i think you missed the one on line 16 lol

Reply:
Whoops. Fixed.